packages/expo-updates/src/Updates.types.ts

```diff
@@ -84,6 +84,54 @@ export type UpdateEvent = {
   message?: string;
 };
 
+/**
+ * An object representing a single log entry from expo-updates logging on the client.
+ */
+export type UpdatesLogEntry = {
+  /**
+   * The time the log was written, in seconds since Jan 1 1970 UTC.
```

Comment:
I think we should make this milliseconds instead of seconds, to keep it consistent with how JS formats timestamps by default

Reply:
Ok

Reply:
Done